tests/openebs-pool-container-failure_test.go

```diff
@@ -166,4 +203,74 @@ var _ = Describe("BDD of openebs pool container failure experiment", func() {
 		})
 	})
 
+	//Matching the Resource Verison after Chaos
+	Context("Check Resource Version of pool container", func() {
+
+		It("Should check for the change in Resource Version after Chaos", func() {
+			rv2 := 0
+			csp_rv, err := client.CoreV1().Pods("openebs").List(metav1.ListOptions{LabelSelector: "app=cstor-pool"})
+			Expect(err).To(BeNil(), "fail to get the csp pods")
+			for _, v := range csp_rv.Items {
+				i2, _ := strconv.Atoi(v.ResourceVersion)
+				rv2 = rv2 + i2
+			}
+
+			if rv1 != rv2 {
+				fmt.Printf("The Resource Version is changes!!!\n")
+			} else {
+				Fail("The Resource Version does not change")
+			}
+
+		})
+	})
+
+	//Matching the ContainerIDs after Chaos
+	Context("Check ContainerIDs after chaos", func() {
+
+		It("Should check for the change in ContainerIDs of csp pod", func() {
+			var container_id2 [9]string
+			l := 0
+			c := 0
+			csp, err := client.CoreV1().Pods("openebs").List(metav1.ListOptions{LabelSelector: "app=cstor-pool"})
+			Expect(err).To(BeNil(), "fail to get the csp pods")
+			for _, v1 := range csp.Items {
+				for i := 0; i < 3; i++ {
+					container_id2[l] = (v1.Status.ContainerStatuses[i].ContainerID)
+					l++
+				}
+			}
+			for i := range container_id1 {
+				if container_id1[i] != container_id2[i] {
+					c = 1
+				}
+			}
+			if c == 0 {
+				Fail("The Container ID does not change")
+			} else {
+				fmt.Printf("Container ID Changes!!!\n")
+			}
+		})
+	})
+
+	//Matching the Container Restart Count after Chaos
+	Context("Check Container Restart Count", func() {
+
+		It("Should check for the change in Container Restart Count after Chaos", func() {
+			rc2 := 0
+			csp_rc, err := client.CoreV1().Pods("openebs").List(metav1.ListOptions{LabelSelector: "app=cstor-pool"})
+			Expect(err).To(BeNil(), "fail to get the csp pods")
+			for _, v := range csp_rc.Items {
+				for i := 0; i < 3; i++ {
+					rc2 = rc2 + int(v.Status.ContainerStatuses[i].RestartCount)
+				}
+			}
+
+			if rc1 != rc2 {
```

Comment:
Instead of writing if-else you can use something like Expect(rc1-rc2).NotTo(Equal(0)). Use better name instead of rc1 & rc2 ( before, after)